Add lesson start and end hour

witekmikolajczak opened the pull request:
![index](https://user-images.githubusercontent.com/61797939/76004499-dc161300-5f09-11ea-8bd8-e2a02ee4830a.jpeg)

witekmikolajczak committed:
Add files via upload

witekmikolajczak committed:
Update bobo.py

Update: Reading hours had been added for the day.

pniedzwiedzinski commented on lesson.py:
```diff
@@ -0,0 +1,42 @@
+start_hour = get_lessons_start()
+end_hour = get_lessons_end()
+
+print ("Dzisiaj lekcje trwają od " + (LESSON_HOURS[start_hour]["start"]) + " do " + (LESSON_HOURS[end_hour]["end"]))
```
As we use these functions in `bobo.py`, this section is not needed.
```suggestion

```

pniedzwiedzinski commented on bobo.py:
```diff
@@ -48,8 +49,10 @@ def main():
-
+    start_hour = get_lesson_start()
+    end_hour = get_lesson_end    
+    send_msg("Dzisiaj lekcje trwają od " + (LESSON_HOURS[start_hour]["start"]) + " do " + (LESSON_HOURS[end_hour]["end"])
```
You forgot to close the braces :wink: 
```suggestion
    send_msg("Dzisiaj lekcje trwają od " + (LESSON_HOURS[start_hour]["start"]) + " do " + (LESSON_HOURS[end_hour]["end"]))
```

pniedzwiedzinski commented on bobo.py:
```diff
@@ -5,6 +5,7 @@
 from datetime import datetime
 
 from holidays import get_today_holiday
+from lessons import get_lesson_start, get_lesson_end
```
The file is called `lesson.py` therefore you should import from `lesson`, not `lessons`. Probably a typo but I want to make sure you get it right. And the functions are in a plural form. 

In line 54 you are using `LESSON_HOURS` but it's also needed to be imported
```suggestion
from lesson import get_lessons_start, get_lessons_end, LESSON_HOURS
```

pniedzwiedzinski commented on bobo.py:
```diff
@@ -48,8 +49,10 @@ def main():
-    send_msg(f"<i>Dzisiaj będzie {len(get_lessons())} lekcji</i>")
-
+    start_hour = get_lesson_start()
+    end_hour = get_lesson_end    
```
To make it work you should call this function :smile_cat: 
```
    end_hour = get_lesson_end()
```

pniedzwiedzinski reviewed (changes requested):
I made it work but it needs a few changes. How I tested it:

0. I installed dependencies `pip install requests`
1. I commented the 11th and 12th line (that are getting the token and chat id from system)
2. I renamed `send_msg` (line 34) to `literally_anything_send_msg`
3. I created this simple function:
```py
def send_msg(m):
    print(m)
```
4. After these changes I can now run the `python bobo.py` and instead of sending, it will print the messages to the console. 

This process is a little bit complicated but that's the fastest way right now to test it. I will probably think of a better solution in the future :laughing:

witekmikolajczak committed:
Update bobo.py

Co-Authored-By: Patryk Niedźwiedziński <pniedzwiedzinski19@gmail.com>

witekmikolajczak committed:
Update bobo.py

Co-Authored-By: Patryk Niedźwiedziński <pniedzwiedzinski19@gmail.com>

witekmikolajczak committed:
Update lesson.py

Co-Authored-By: Patryk Niedźwiedziński <pniedzwiedzinski19@gmail.com>

witekmikolajczak committed:
Update bobo.py final version

I have looked at this code for 10 minutes and i couldn't notice that i had bad called functions in 52/53 line
Now it should work

pniedzwiedzinski reviewed (approved):
Now it's perfect!
